ParabolInc/parabol
# Jira Server: show integrated task

Resolves #5994

Needs #6059

![image](https://user-images.githubusercontent.com/466991/155158609-99a44731-9944-4dda-a2df-ff7bb211a6e6.png)


**How to test:**

- Add Jira Server integration
- Create a task in parabol
- Using push to integration menu, push a task to the selected Jira Server
- See the task was pushed successfully
- Open task in Jira Server, see it was created correctly
- Edit task in Jira Server
- Open parabol board, see the task is loaded correctly
